public/components/mainMenu.js

```diff
@@ -45,17 +45,36 @@ const MenuTemplate = (window) => {
     {
       label: 'View',
       submenu: [
-        { role: 'reload' },
-        { role: 'forcereload' },
-        { role: 'toggledevtools' },
-        { type: 'separator' },
         { role: 'resetzoom' },
         { role: 'zoomin' },
         { role: 'zoomout' },
         { type: 'separator' },
         { role: 'togglefullscreen' },
       ],
     },
+    {
+      label: 'Develop',
+      submenu: [
+        { role: 'reload' },
+        { role: 'forcereload' },
+        { role: 'toggledevtools' },
+        { type: 'separator' },
+        {
+          label: 'Clear storage data',
+          click: () => {
+            dialog.showMessageBox({
+              message: 'This will reset all app data, are you sure?',
+              buttons: ['OK', 'Cancel'],
+            }, (response) => {
+              if (response === 0) {
+                window.webContents.session.clearStorageData();
```

Comment:
I just noticed that [this is async & takes a callback](https://electronjs.org/docs/api/session#sesclearstoragedataoptions-callback). This seems to work as-is, but might want to move reload into the callback to be safe?

Reply:
Good spot, thanks!